kratos/input_output/vtk_output.cpp

```diff
@@ -0,0 +1,452 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Aditya Ghantasala 
+//
+//
+
+// project includes
+#include "vtk_output.h"
+#include <typeinfo>
+
+namespace Kratos
+{
+
+VtkOutput::VtkOutput(ModelPart &rModelPart, Parameters rParameters) : mrModelPart(rModelPart), mrOutputSettings(rParameters)
+{
+    mDefaultPrecision = this->mrOutputSettings["output_precision"].GetInt();;
+    this->mDoneTest = false;
+    this->mShouldSwap = false;
+    std::string file_format = this->mrOutputSettings["file_format"].GetString();
+    if(file_format == "ascii")
+        mFileFormat =  VtkOutput::FileFormat::VTK_ASCII;
+    else if (file_format == "binary")
+        mFileFormat = VtkOutput::FileFormat::VTK_BINARY;
+    else
+        KRATOS_ERROR<<"Option for file_format : "<<file_format<<" not recognised.!"<<std::endl
+            <<"Possible output formats options for VTKOutput are :: ascii and binary "<<std::endl;
+}
+
+VtkOutput::~VtkOutput(){};
+
+
+void VtkOutput::CreateMapFromKratosIdToVTKId(const ModelPart &rModelPart)
+{
+    int vtk_id = 0;
+    for(const auto& node : rModelPart.Nodes())
+    {
+        int KratosId = node.Id();
+        mKratosIdToVtkId[KratosId] = vtk_id;
+        vtk_id++;
```

Comment:
I think those three lines can be written as one line:
~~~
mKratosIdToVtkId[node.Id()] = vtk_id++;
~~~